src/components/CalendarMonthGrid.jsx

```diff
@@ -166,6 +187,18 @@ export default class CalendarMonthGrid extends React.Component {
     this.container = ref;
   }
 
+  setMonthHeight(height, i) {
+    if (this.calendarMonthHeights[i]) {
+      if (i === 0) {
+        this.calendarMonthHeights = [height, ...this.calendarMonthHeights.slice(0, -1)];
+      } else if (i === this.calendarMonthHeights.length - 1) {
+        this.calendarMonthHeights = [...this.calendarMonthHeights.slice(1), height];
```

Comment:
it's slightly more efficient here to do `this.calendarMonthHeights.slice(1).concat(height)`